modules_basic/compose.js

```diff
@@ -47,6 +48,22 @@ exports.create = function (api) {
       placeholder: opts.placeholder || 'Write a message'
     })
 
+    var channelInput = h('input.channel', {
+      placeholder: '#channel',
+      value: meta.channel ? `#${meta.channel}` : '',
+      disabled: meta.channel ? true : false
+    })
+
+    channelInput.addEventListener('keydown', function (e) {
```

Comment:
i'm going to bring some of this into style alignment (I've been moving more anonymous functions like this to fat arrow functions), and remove the console.log.

Happy to merge this and make that change.

awesome work ! :taco: